public/app/api/README.md

```diff
@@ -0,0 +1,160 @@
+## Generating RTK Query API Clients
+
+To show the steps to follow, we are going to work on adding an API client to create a new dashboard. Just adapt the following guide to your use case.
+
+### 1. Generate an OpenAPI snapshot
+
+First, check if the `group` and the `version` are already present in [openapi_test.go](/pkg/tests/apis/openapi_test.go). If so, move on to the next step.
+<br/> If you need to add a new block, you can check for the right `group` and `version` in the backend API call that you want to replicate in the frontend.
+
+```jsx
+{
+    Group:   "dashboard.grafana.app",
+	Version: "v0alpha1",
+}
+```
+
+### 2. Run the `TestIntegrationOpenAPIs` test
+
+Note that it will fail the first time you run it, and, in the second one, it will generate the corresponding OpenAPI spec. You will find it in [openapi_snapshots](/pkg/tests/apis/openapi_snapshots).
```

Comment:
```suggestion
Note that it will fail the first time you run it. On the second run, it will generate the corresponding OpenAPI spec, which you can find in [openapi_snapshots](/pkg/tests/apis/openapi_snapshots).
```